tests/extensions/test_common.py

```diff
@@ -0,0 +1,58 @@
+from a2a.extensions.common import (
+    find_extension_by_uri,
+    get_requested_extensions,
+)
+from a2a.types import AgentCapabilities, AgentCard, AgentExtension
+
+
+def test_get_requested_extensions():
+    assert get_requested_extensions([]) == set()
+    assert get_requested_extensions(['foo']) == {'foo'}
+    assert get_requested_extensions(['foo', 'bar']) == {'foo', 'bar'}
+    assert get_requested_extensions(['foo, bar']) == {'foo', 'bar'}
+    assert get_requested_extensions(['foo,bar']) == {'foo', 'bar'}
+    assert get_requested_extensions(['foo', 'bar,baz']) == {'foo', 'bar', 'baz'}
+    assert get_requested_extensions(['foo,, bar', 'baz']) == {
+        'foo',
+        'bar',
+        'baz',
+    }
+    assert get_requested_extensions([' foo , bar ', 'baz']) == {
+        'foo',
+        'bar',
+        'baz',
+    }
+
+
+def test_find_extension_by_uri():
+    ext1 = AgentExtension(uri='foo', description='The Foo extension')
+    ext2 = AgentExtension(uri='bar', description='The Bar extension')
+    card = AgentCard(
+        name='Test Agent',
+        description='Test Agent Description',
+        version='1.0',
+        url='http://test.com',
+        skills=[],
+        defaultInputModes=['text/plain'],
+        defaultOutputModes=['text/plain'],
```

Comment:
convert all of the camelCase fields to snake_case

Reply:
Done.